### PR TITLE
proper paper references+citations

### PR DESCRIPTION
**Pull request by mikkelmilo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +48,39 @@ It supports Liquidity and Elm as target languages. The extraction also features
...
+- ["Verifying, testing and running smart contracts in ConCert"](https://cs.au.dk/fileadmin/site_files/cs/AA_pdf/COBRA_Paper_-_Verifying__testing_and_running_smart_contracts_in_ConCert.pdf)
+  Danil Annenkov, Mikkel Milo, Jakob Botsch Nielsen, Bas Spitters. Coq Workshop 2020.
+### Citing the papers
+@misc{nielsen2019smart,
+      title={Smart Contract Interactions in Coq}, 
+      author={Jakob Botsch Nielsen and Bas Spitters},
+      year={2019},
+      eprint={1911.04732},
+      archivePrefix={arXiv},
+      primaryClass={cs.LO}
```
Comment (spitters) [inline README.md:66]: No need for this line.

File: README.md [hunk 2]
```diff
@@ -50,3 +48,39 @@ It supports Liquidity and Elm as target languages. The extraction also features
 ## Notes for developers
 
 The [execution](execution/) subproject can be built independently via running `make` in the `execution` folder. This also means that the `_CoqProject` file inside the `execution` folder must be manually kept in sync with the main `_CoqProject` in the root.
+
+## Papers
+ - ["Smart Contract Interactions in Coq"](https://arxiv.org/abs/1911.04732)
+   Jakob Botsch Nielsen, Bas Spitters. Presented at 1st Workshop on Formal Methods for Blockchains, 3rd Formal Methods World Congress, October 2019.
```
Comment (spitters) [inline README.md:54]: Remove: presented at

File: README.md [hunk 3]
```diff
@@ -50,3 +48,39 @@ It supports Liquidity and Elm as target languages. The extraction also features
...
+   journal={Proceedings of the 9th ACM SIGPLAN International Conference on Certified Programs and Proofs},
+   publisher={ACM},
+   author={Annenkov, Danil and Nielsen, Jakob Botsch and Spitters, Bas},
+   year={2020},
+   month={Jan}
+}
+
+@inproceedings{Annenkov2020VerifyingTA,
+  title={Verifying, testing and running smart contracts in ConCert},
+  author={D. Annenkov and Mikkel Milo and Jakob Botsch Nielsen and Bas Spitters},
```
Comment (spitters) [inline README.md:83]: D -> Danil

File: README.md (hunk 1 above)
Comment (jakobbotsch) [reply README.md:66]: Also I think there is a better citation available.
```
@inproceedings{DBLP:conf/fm/NielsenS19,
  author    = {Jakob Botsch Nielsen and
               Bas Spitters},
  title     = {Smart Contract Interactions in Coq},
  booktitle = {{FM} Workshops {(1)}},
  series    = {Lecture Notes in Computer Science},
  volume    = {12232},
  pages     = {380--391},
  publisher = {Springer},
  year      = {2019}
}
```

File: README.md (hunk 3 above)
Comment (jakobbotsch) [reply README.md:83]: This one isn't in proceedings, right?

Comment (annenkov) [reply README.md:83]: No, not really. Maybe change it to `@misc`?

**Comment by mikkelmilo** [comment]
How can the build fail on the PR, but not on the push? And how can it even fail when the only thing that changed is the README? (and the previous commit did not fail)...

**Comment by jakobbotsch** [comment]
It seems exit code 137 is due to out of memory from some googling so probably the build is just dangerously close to the memory limit we get with GitHub. If this keeps happening we can make the build nonparallel. I have rerun the builds.

File: README.md (hunk 3 above)
Comment (jakobbotsch) [reply README.md]: It looks like you forgot to address Bas's feedback

File: README.md [hunk 4]
```diff
@@ -50,3 +48,45 @@ It supports Liquidity and Elm as target languages. The extraction also features
...
+  author    = {Jakob Botsch Nielsen and
+               Bas Spitters},
+  title     = {Smart Contract Interactions in Coq},
+  booktitle = {{FM} Workshops {(1)}},
+  series    = {Lecture Notes in Computer Science},
+  volume    = {12232},
+  pages     = {380--391},
+  publisher = {Springer},
+  year      = {2019}
+}
```
Comment (jakobbotsch) [inline README.md:88]:
````suggestion
```
@misc{Annenkov2020VerifyingTA,
  title={Verifying, testing and running smart contracts in ConCert},
  author={D. Annenkov and Mikkel Milo and Jakob Botsch Nielsen and Bas Spitters},
  year={2020}
}

@article{Annenkov_2020,
   title={ConCert: a smart contract certification framework in Coq},
   ISBN={9781450370974},
   url={http://dx.doi.org/10.1145/3372885.3373829},
   DOI={10.1145/3372885.3373829},
   journal={Proceedings of the 9th ACM SIGPLAN International Conference on Certified Programs and Proofs},
   publisher={ACM},
   author={Annenkov, Danil and Nielsen, Jakob Botsch and Spitters, Bas},
   year={2020},
   month={Jan}
}

@inproceedings{smart-contract-interactions,
  author    = {Jakob Botsch Nielsen and
               Bas Spitters},
  title     = {Smart Contract Interactions in Coq},
  booktitle = {{FM} Workshops {(1)}},
  series    = {Lecture Notes in Computer Science},
  volume    = {12232},
  pages     = {380--391},
  publisher = {Springer},
  year      = {2019}
}
```
````

**Comment by jakobbotsch** [comment]
Not sure what CI is stuck on, but anyway, feel free to merge this

**Pull request merged**